internal/ingest/linksystem.go

```diff
@@ -566,7 +567,7 @@ func (ing *Ingester) ingestEntriesFromPublisher(ctx context.Context, ad schema.A
 				return mhCount, err
 			}
 			wrappedErr := fmt.Errorf("failed to sync entries: %w", err)
-			if strings.Contains(err.Error(), "content not found") {
+			if errors.Is(err, ipld.ErrNotExists{}) || strings.Contains(err.Error(), "content not found") {
```

Comment:
It might be worth converting the "content not found" error from graphsync into an `ipld.ErrNotExists{}` error in go-libipni.

Reply:
This should convert the "content not found" error: https://github.com/ipni/go-libipni/pull/40